src/Extension/CoreExtension.php

```diff
@@ -1285,6 +1287,75 @@ public static function testEmpty($value): bool
         return '' === $value || false === $value || null === $value || [] === $value;
     }
 
+    /**
+     * Checks if a variable is traversable.
+     *
+     *    {# evaluates to true if the foo variable is an array or a traversable object #}
+     *    {% if foo is iterable %}
+     *        {# ... #}
+     *    {% endif %}
+     *
+     * @param mixed $value A variable
+     *
+     * @return bool true if the value is traversable
+     *
+     * @deprecated since Twig 3.8, to be removed in 4.0 (use the native "is_iterable" function instead)
+     *
+     * @internal
+     */
+    public static function testIterable($value)
+    {
+        return is_iterable($value);
+    }
+
+    /**
+     * Checks if a variable is a sequence.
+     *
+     * Traversables must be converted before being tested.
+     *
+     *    {# evaluates to true if the foo variable is a sequence #}
+     *    {% if foo is sequence %}
+     *        {# ... #}
+     *    {% endif %}
+     *
+     * @param mixed $value A variable
+     *
+     * @return bool true if the value is a sequence
+     */
+    public static function testSequence($value)
+    {
+        if ($value instanceof \ArrayObject) {
+            $value = $value->getArrayCopy();
+        }
+
+        // array_is_list() is provided by symfony/polyfill-php81.
+        return \is_array($value) && array_is_list($value);
+    }
+
+    /**
+     * Checks if a variable is a mapping.
+     *
+     * Traversables must be converted before being tested.
+     *
+     *    {# evaluates to true if the foo variable is a mapping #}
+     *    {% if foo is mapping %}
+     *        {# ... #}
+     *    {% endif %}
+     *
+     * @param mixed $value A variable
+     *
+     * @return bool true if the value is a mapping
+     */
+    public static function testMapping($value)
+    {
+        if ($value instanceof \ArrayObject) {
+            $value = $value->getArrayCopy();
+        }
+
+        // array_is_list() is provided by symfony/polyfill-php81.
```

Comment:
I suggest removing this comment entirely (for PHP 8.1+, the function is provided by PHP itself, so the comment is not even correct)

Reply:
Done in https://github.com/twigphp/Twig/pull/3859/commits/3d23fe1c1aaca508021e4ca2f372e8bdccd648aa